src/app/dashboard/Missions/MissionEditView.jsx

```diff
@@ -380,6 +374,7 @@ const MissionEditView = ({ mission, toDetailsView, toListView, volunteers }) =>
             </Card>
             <Card label="Delivery Details" classes={classes}>
               <Row Icon={ScheduleIcon} classes={classes}>
+                {" "}
```

Comment:
If we're trying to add spacing to both sides of this input, can we use CSS instead?

Reply:
Oops, not sure how that got in there, but it's gone now.